.github/pull_request_template.md

```diff
@@ -1,26 +1,79 @@
-### This pull request changes...
+### Summary of Changes
+_Provide a brief summary of changes_
+_Addresses issue #_
+_Acceptance criteria as stated in the issue_
 
-*List expected changes*
+### How to Test
+_List the steps to test the PR_
 
+### Deliverable 1: Accepted Features
 
-## FUNCTIONALITY
-*Describe the functionality*
-*Addresses Issue #*
+> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*
 
+> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*
 
-## TO TEST
++ [ ] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
++ [ ] Check against the criteria:
 
-*List the steps to test the PR*
+As facilitator/product manager, @kniz-raft will decide if ACs are met.
```

Comment:
will @lfrohlich also be involved in this decision point? the QASP checklist currently notes that product owner decides if ACs are met.

Reply:
@ADPennington Ultimately, yes, but this is for when we put the pull request in and approve it on our side. When it goes into QASP Review, you will post your own QASP checklist and that is where the product owner makes the ultimate decision.

Reply:
Maybe we can clarify by saying "if ACs are met before being moved from Raft to QASP review."?

Reply:
That would work. Or we could say, "...will decide if ACs are met from raft's perspective"?

Reply:
@carltonsmith thanks! good with either suggestion from you or @lfrohlich